de1plus/plugins/example/plugin.tcl

```diff
@@ -1,21 +1,52 @@
+
+# Change package name for you extension / plugin
 set plugin_name "example"
 
+# These are chown in the plugin selection page
 set ::plugins::${plugin_name}::author "JoJo"
 set ::plugins::${plugin_name}::contact "email@coffee-mail.de"
 set ::plugins::${plugin_name}::version 1.0
-set ::plugins::${plugin_name}::description "Minimal plugin to showcase the interface"
+set ::plugins::${plugin_name}::description "Minimal plugin to showcase the interface of the plugin / extensions system."
+
+# only set if you want to provide a settings page for your plugin
+# returns the name of the page to show next
+proc ::plugins::${plugin_name}::settingsfunction {} {
+    # Unique name per page
+    set page_name "plugin_example_page_default"
+
+    # Background image and "Done" button
+    add_de1_page "$page_name" "settings_message.png" "default"
+    add_de1_text $page_name 1280 1310 -text [translate "Done"] -font Helv_10_bold -fill "#fAfBff" -anchor "center"
+	add_de1_button $page_name {say [translate {Done}] $::settings(sound_button_in); fill_extensions_listbox; page_to_show_when_off extensions; set_extensions_scrollbar_dimensions}  980 1210 1580 1410 ""
```

Comment:
Do I need to something special to go back?

Reply:
Are you dynamically creating new pages?  I've not done that, I create them all when the app launches.

If not, then I recommend you look at the log (make sure it's being created) as page_to_show_when_off will automatically hide elements that are not part of that context and you can confirm that's the case.  Make sure in the log that page to show / page to hide, is what you expect.

There's not enough code above for me to guess what the issue is, and using $page_name is hiding the context info from me, so I can't really review it.